Introduction.md

```diff
@@ -129,6 +113,50 @@ There is a script to update a WebKit checkout: `Tools/Scripts/update-webkit`. Th
 
 [See Building WebKit](https://github.com/WebKit/webkit/blob/master/ReadMe.md#building-webkit)
 
+### Getting setup to contribute
+
+If you've followed the steps above, get setup to contribute by running:
+
+```Bash
+git webkit setup
+```
+
+The `setup` checks that your environment is optimally configured to contribute, and may prompt you for some additional information.
+
+### Submitting a pull request
+
+Firstly, please make sure you [file a bug](https://bugs.webkit.org) for the thing you are adding or fixing! Or, find a bug that you think is relevant to the fix you are making.
+
+Assuming you are working off "main" branch, once your patch is working and [tests are passing](#correctness-testing-in-webkit), simply run:
+
+```Bash
+git webkit pr --issue <your bug number here>
+```
+
+That will pull down the details from [bugs.webkit.org](https://bugs.webkit.org), create a new git branch, and generate a commit message for you.
+You shouldn't need to make any changes to the auto-generated commit message.
```

Comment:
This isn't true, right? We expect contributors to add descriptions of what they are changing into their commit message.

Reply:
Ah, I've never done that for any patches as I was scared of breaking the auto-generated commit message... or that it would get trashed after making changes. The old changelogs files were a bit temperamental, so I always trashed them and regenerated them.   

Let me try it by fixing this sentence...